nx/blocks/snapshot-admin/utils/utils.js

```diff
@@ -165,3 +165,38 @@ export async function copyManifest(name, resources, direction) {
   const queue = new Queue(copyUrl, 50);
   await Promise.all(urls.map((url) => queue.push(url)));
 }
+
+export async function updateScheduledPublish(snapshotId) {
+  const adminURL = `${SNAPSHOT_SCHEDULER_URL}/schedule`;
+  const body = {
+    org,
+    site,
+    snapshotId,
+  };
+  const headers = { 'content-type': 'application/json' };
+  const resp = await fetch(`${adminURL}`, {
+    method: 'POST',
+    headers,
+    body: JSON.stringify(body),
+  });
+  const result = resp.headers.get('X-Error');
+  return { status: resp.status, text: result };
+}
+
+export async function isRegisteredForSnapshotScheduler() {
+  try {
+    const adminURL = `${SNAPSHOT_SCHEDULER_URL}/register/${org}/${site}`;
+    const resp = await fetch(adminURL);
+    return resp.status === 200;
+  } catch (error) {
+    console.error('Error checking if registered for snapshot scheduler', error);
+    return false;
+  }
+}
+
+// Convert UTC date to local datetime-local format
+export function formatLocalDate(utcDate) {
+  if (!utcDate) return '';
+  const d = new Date(utcDate);
+  return `${d.getFullYear()}-${String(d.getMonth() + 1).padStart(2, '0')}-${String(d.getDate()).padStart(2, '0')}T${String(d.getHours()).padStart(2, '0')}:${String(d.getMinutes()).padStart(2, '0')}`;
+}
```

Comment:
We have functions that already do this in our preferred formats that are also international friendly.

```
export function formatDate(timestamp) {
  const rawDate = timestamp ? new Date(timestamp) : new Date();
  const date = rawDate.toLocaleDateString([], { year: 'numeric', month: 'short', day: 'numeric' });
  const time = rawDate.toLocaleTimeString([], { hour: 'numeric', minute: '2-digit' });
  return { date, time };
}
```